test/extensions/listener_managers/listener_manager/listener_manager_impl_test.cc

```diff
@@ -3339,12 +3338,12 @@ name: foo
   EXPECT_CALL(*worker_, stopListener(_, _, _));
   EXPECT_CALL(*listener_factory_.socket_, close());
   EXPECT_CALL(*listener_foo, onDestroy());
-  Network::ExtraShutdownListenerOptions empty_options;
-  manager_->stopListeners(ListenerManager::StopListenersType::All, empty_options);
-  EXPECT_EQ(1, server_.stats_store_.counterFromString("listener_manager.listener_stopped").value());
+  manager_->stopListeners(ListenerManager::StopListenersType::All, {);
+    EXPECT_EQ(1,
+              server_.stats_store_.counterFromString("listener_manager.listener_stopped").value());
 
-  // Validate that adding a listener is not allowed after all listeners are stopped.
-  const std::string listener_bar_yaml = R"EOF(
```

Comment:
Something wrong with the formatting in this file?

Reply:
Doh, typo.